src/components/StateChannelsTableCondensed.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="state-channels-table-condensed">
+  <div>
```

Comment:
is the div needed in such cases?

Reply:
yes it is, as the class is applied from the parent component